mypy_django_plugin/transformers/fields.py

```diff
@@ -93,11 +93,12 @@ def fill_descriptor_types_for_related_field(ctx: FunctionContext, django_context
         related_model_type = Instance(related_model_info, [])  # type: ignore
 
     related_model_to_set_info = helpers.lookup_class_typeinfo(typechecker_api, related_model_to_set)
+    related_model_to_set_type: Union[MypyType, Instance]
```

Comment:
```suggestion
    related_model_to_set_type: MypyType
```

Since Instance is a type